app/views/templates/_news_and_documents.html.erb

```diff
@@ -26,13 +26,17 @@
         if articles.any? %>
           <articles>
 
-            <% articles.each do |a| %>
+            <% articles.each do |article| %>
+
+              <article-listing category="<%= cms_block_content(:article_type, article) %>" ref="article">
+                <h3><%= link_to article.label, article.url(:relative) %></h3>
+
+                <% unless cms_block_content(:image, article).to_s.strip.empty? %>
+                  image_tag cms_block_content(:image, article).file.url(:small), alt: article.label
```

Comment:
This will still need a `<%=    %>` tag around it as it is embedded ruby in a view